drv/OpenHashSet.drv

```diff
@@ -569,15 +569,73 @@ public class OPEN_HASH_SET KEY_GENERIC extends ABSTRACT_SET KEY_GENERIC implemen
 		this(a, DEFAULT_LOAD_FACTOR);
 	}
 
+	/** Creates new empty hash set. */
+	public static KEY_GENERIC OPEN_HASH_SET KEY_GENERIC of() {
```

Comment:
general suggestion, just came to mind: I was a bit confused for a second, seeing `of` here as returning mutable types. Maybe we could have the convention of saying `create` (akin to Guava) for mutable factory methods, `of` for immutable creation, `copyOf` for immutable copy?

Reply:
Well, my list implementations expose the underlying array, so copying does not change much the situation. As in the case of Set, we would need an actual immutable implementation. I will commit soon something to that purpose.